src/charm.py

```diff
@@ -107,7 +107,7 @@ def change_config(self) -> None:
             return
         self.model.unit.status = ops.MaintenanceStatus("Configuring Synapse")
         try:
-            self.pebble_service.change_config(container)
+            self.pebble_service.change_config(container=container)
```

Comment:
Just out of curiosity, why the change here, is it purely cosmetic?

Reply:
My bad, I changed this while trying the idea of passing the token as a parameter and then left as a named parameter. Reverted, thanks!